package.json

```diff
@@ -46,7 +46,8 @@
     "babel-traverse": "^6.26.0",
     "babylon": "^6.18.0",
     "lodash.isplainobject": "^4.0.6",
+    "promise-synchronizer": "^1.0.6",
```

Comment:
this is not a package i'm interested in using; making async things async is a very dangerous choice to make in JS.